packages/superset-ui-color/src/cleanValue.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Ensure value is a string
+ * @param {any} value
+ */
+export default function cleanValue(value) {
```

Comment:
could we possibly make this have a more specific name? `clean` is pretty generic, something like `stringyifyAndTrim` would make it more readable/clear what exactly "clean" means.